proto/global_admin.proto

```diff
@@ -10,6 +10,10 @@ package global_admin;
 service GlobalAdmin {
   rpc GetAccounts(_GetAccountsRequest) returns(_GetAccountsResponse) {}
   rpc GetAccountSessionToken (_GetAccountSessionTokenRequest) returns(_GetAccountSessionTokenResponse) {}
+  rpc AddMember(_AddMemberRequest) returns(_AddMemberResponse) {}
+  rpc RemoveMember(_RemoveMemberRequest) returns(_RemoveMemberResponse) {}
+  rpc ListMembers(_ListMembersRequest) returns(_ListMembersResponse) {}
+  rpc GetEndpointsForAccount(_GetEndpointsForAccountRequest) returns (_GetEndpointsForAccountResponse) {}
```

Comment:
@rlinehan jfyi I think `GetEndpointsForAccount` is also good to go into client-protos repo?

Reply:
yes!